android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -871,6 +871,18 @@ abstract class EngineSession(
         onException: (Throwable) -> Unit,
     )
 
+    /**
+     * Requests the creation of product analysis for a given product page URL.
+     *
+     * @param onSuccess callback invoked if the engine API returns a valid response.
+     * @param onError callback invoked if there was an error getting the response.
+     */
+    abstract fun requestCreateProductAnalysis(
```

Comment:
AFAIU this is the analyze API, should we name it `analyzeProduct` or `reAnalyzeProduct` here so it's more clear?

Reply:
renamed to reanalyzeProduct